document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -1167,7 +1172,7 @@ private String getColumnValue(
       }
 
       // check for ID column
```

Comment:
```suggestion
      // check for any OUTER column including ID, and its alias
```

Reply:
If, we need this - https://github.com/hypertrace/document-store/pull/132#discussion_r984253529, let's change the comment

Reply:
Done.